docs/ops.html

```diff
@@ -3992,6 +3979,75 @@ <h3>Finalizing the migration</h3>
 
 # Other configs ...</pre>
 
+  <h3>Reverting to ZooKeeper mode During the Migration</h3>
+  <p>
+    While the cluster is still in migration mode, it is possible to revert to ZK mode.  The process
+    to follow depends on how far the migration has progressed. In order to find out how to revert,
```

Comment:
Should we explain on which phase we could revert to ZK mode? Or at least say after the first roll?

Reply:
It's explained by the table, I think.